security/index.adoc

```diff
@@ -0,0 +1,90 @@
+[id="security-compliance-overview"]
+= {product-title} security and compliance
+include::modules/common-attributes.adoc[]
+:context: security-compliance-overview
+
+toc::[]
+
+// TODO: Pull out things that don't apply for previous versions
+
+// TODO: Need some help with an intro blurb
+
+[id="security-overview"]
+== Security overview
+
+It is important to understand how to properly secure various aspects of your {product-title} cluster.
+
+[discrete]
+=== Container security
+
+A good starting point to understanding {product-title} security is to review the concepts in xref:../security/container_security/security-understanding.adoc#security-understanding[Understanding container security]. This and subsequent sections provide a high-level walkthrough of the container security measures available in {product-title}, including solutions for the host layer, the container and orchestration layer, and the build and application layer. These sections also include information on:
+
+* Why container security is important and how it compares with existing security standards.
+* Which container security measures are provided by the host ({op-system} and {op-system-base}) layer and
+which are provided by {product-title}.
+* How to evaluate your container content and sources for vulnerabilities.
+* How to design your build and deployment process to proactively check container content.
+* How to control access to containers through authentication and authorization.
+* How networking and attached storage are secured in {product-title}.
+* Containerized solutions for API management and SSO.
+
+[discrete]
+=== Auditing
+
+Audit provides a security-relevant chronological set of records documenting the sequence of activities that have affected the system by individual users, administrators, or other components of the system. Administrators can xref:../security/audit-log-policy-config.adoc#audit-log-policy-config[configure the audit log policy] and xref:../security/audit-log-view.adoc#audit-log-view[view audit logs].
```

Comment:
The sentence starts as "Audit provides a security-relevant..." Should "Audit" be changed to Auditing" or something else?

Reply:
Yeah, I pulled that wording from another file in the existing docs. I thought it sounded weird too. I don't think it should be "Auditing" though, so I'll have to give it some thought

Reply:
@TanyaShearonRH Okay so I changed it to "{product-title} auditing" (will resolve to "OpenShift Container Platform auditing")